docs/specification/auth/_index.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Authentication
+type: docs 
+weight: 50
+---
+
+{{< callout type="warning" >}}
+Auth is **experimental**, and being drafted for release in the next [revision]({{< ref "/specification/revisions" >}}) of the protocol.
+
+The additions to the base protocol are backwards compatible to revision 2024-11-05; however, **the auth specification may change in backwards incompatible ways** until the next protocol revision.
+{{< /callout >}}
+
+The Model Context Protocol (MCP) provides a standardized way for clients and servers to establish secure communication channels. The authentication framework supports multiple authentication methods to accommodate different security requirements and use cases.
+
+## User Interaction Model
+
+Authentication in MCP is designed to be flexible and secure, supporting various authentication flows that may require different levels of user interaction:
+
+- OAuth 2.0 for standardized authorization flows
+- Credential-based authentication for API keys and similar secrets
+
+Other authentication schemes may also be implemented by clients and servers as extensions to the base protocol.
+
+## Capabilities
+
+Clients that support authentication **MUST** declare their supported authentication methods during [initialization]({{< ref "/specification/basic/lifecycle#initialization" >}}):
+
+```json
+{
+  "capabilities": {
+    "auth": {
+      "oauth2": true,
+      "credential": true,
```

Comment:
Generally we have refrained from using booleans here. The existence of the key in the object is implicitly declaring the intent that it's enabled. This gives us the opportunity of having an open object for the key that cna be used in the future for more nuanced declarations.
```suggestion
      "oauth2": {},
      "credential": {},
```
Notable LSP had a bunch of boolean fields and later required more details and ended up with duplicated entries or slightly annoying "can be boolean or an object" (`bool | Params`), which are fine in Typescript but rather annoying to handle in some other languages.